orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -113,32 +111,27 @@ def _extract_input_information(module, *inputs, **kwargs):
 class ORTModule(torch.nn.Module):
 
     def __init__(self, module):
-        assert isinstance(module, torch.nn.Module), "'module' mst be a torch.nn.Module"
+        assert isinstance(module, torch.nn.Module), "'module' must be a torch.nn.Module"
         super(ORTModule, self).__init__()
 
-        self._export_again = False
         # TODO: This is incorrect when different layers may be in different devices
         self._device = next(module.parameters()).device
         self._device_changed = False
 
         # User module is wrapped to use its initializers and save computed gradients
         self._original_module = module
         self._onnx_training = None
+        self._is_training = True
```

Comment:
What are you're thoughts on using the ```_original_module``` variable for checking whether or not the training mode is enabled?

Reply:
Could you please elaborate? 
I am not sure how to use _original_module  to indicate the training_mode ...

Reply:
Just check the boolean ```self._original_module.training``` boolean flag?